Lib/test/test_ctypes/test_arrays.py

```diff
@@ -14,6 +14,10 @@
 formats = c_byte, c_ubyte, c_short, c_ushort, c_int, c_uint, \
           c_long, c_ulonglong, c_float, c_double, c_longdouble
 
+ArrayType = type(Array)
+# _CData is not exported to Python, so we have to access it from __base__.
+_CData = Structure.__base__
+
```

Comment:
How about instead adding a new `TestCase` for the type tests, and move these globals into that class? What do you think?

Reply:
I'm not sure if I fully understand what you mean. Did you mean to add a new test file and group all the related tests in it?

Reply:
Sorry for being unclear; yes, that's what I meant. That would also not pollute the global namespace in this file. I'm not sure it is worth it; what you've got is also fine.

Reply:
If the concern is the global namespace pollution, I think we can add a `support.py` like other test modules to create the `_CData` and other `XxxTypes`.

Reply:
Sounds good to me!